src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -2244,6 +2245,8 @@ public static int Main()
     }
 }";
             var compilation = CreateCompilation(text);
+            compilation.GetDiagnostics();
```

Comment:
>compilation.GetDiagnostics(); [](start = 12, length = 29)

Instead of modifying this particular unit-test, consider creating a new one. It looks like the original test is targeting specific sequence of API calls.